.clj-kondo/taoensso/encore/taoensso/encore.clj

```diff
@@ -0,0 +1,16 @@
+(ns taoensso.encore
```

Comment:
could you pls elaborate on how this related to PR?

Reply:
This is not related to the PR. 

These files need to be committed as these are generated by encore clj-kondo.
https://discord.com/channels/1103692771585433630/1103692773363810317/1166397710941814794

I can remove these changes if not required.